### PR TITLE
Flyway integration

### PR DESCRIPTION
**Pull request by golldfish**

### CODE REVIEW

**Comment by MaciejJ0** [comment]
I would change the file format from "application.properties" to "application.yaml"

**Comment by golldfish** [comment]
> I would change the file format from "application.properties" to "application.yaml"

Why? Could you provide some advantages of this solution please?

File: src/main/java/com/intive/patronage22/szczecin/retroboard/configuration/flyway.conf [hunk 1]
```diff
@@ -0,0 +1,5 @@
+flyway.user=admin
```
Comment (patrykzawadzkiintive) [inline src/main/java/com/intive/patronage22/szczecin/retroboard/configuration/flyway.conf:1]: Can you move properties to application.properties 
spring.flyway.baseline-on-migrate = true
spring.flyway.encoding = UTF-8
spring.flyway.url  = ${spring.datasource.url}
spring.flyway.user = ${spring.datasource.username}
spring.flyway.password = ${spring.datasource.password}
spring.flyway.table=schema_version
spring.flyway.clean-disabled = true

Then remove that config file.

File: pom.xml [hunk 2]
```diff
@@ -15,12 +15,13 @@
 	<description>Patronage 2022 - Java Szczecin - Retrospective board tool</description>
 	<properties>
 		<java.version>11</java.version>
+		<flyway.configFiles>src/main/java/com/intive/patronage22/szczecin/retroboard/configuration/flyway.conf</flyway.configFiles>
```
Comment (patrykzawadzkiintive) [inline pom.xml:18]: This line could be removed if config is in properties file.

**Review by patrykzawadzkiintive — CHANGES_REQUESTED** [review]

**Comment by patrykzawadzkiintive** [comment]
> 

It's up to you how you store properties, whether .properties or .yaml but keep on solution for whole project.

File: README.md [hunk 3]
```diff
@@ -7,3 +7,14 @@
 docker build -t retroboard .
 docker run -p 8080:8080 --rm -it retroboard:latest
 ```
+
+###Flyway
```
Comment (daniel-kmiecik) [inline README.md:11]: There is no need to do this commands manually. Spring Boot will automatically find the script and run it when the application starts.

**Review by MaciejJ0 — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (golldfish) [reply README.md:11]: Thanks, I will remove them in next fix.

**Review by Szachista — APPROVED** [review]

**Review by daniel-kmiecik — APPROVED** [review]

**Review by edych — APPROVED** [review]

**Review by patrykzawadzkiintive — APPROVED** [review]

**Pull request merged**